### PR TITLE
 Abductor additions: replaces and improves glands, new surgery type 

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: Abductors now have the chemical, electric and trauma glands.
del: Removed the bloody, bodysnatch and EMP glands.
tweak: The abductor viral gland now generates a random advanced virus instead of a basic one.
tweak: The abductor heal gland now also heals toxin damage.
tweak: The abductor mindshock gland can now cause different reactions.
tweak: The abductor slime gland now gives the slime faction.
tweak: The abductor species gland now randomizes the victim's appearance and name on top of the species.
add: Abductors have a new surgery type: violence neutralization. It has the same steps as brain surgery, but it will instead inflict a Pacifism trauma upon the victim, making them mostly harmless in the future.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/abductor/equipment/gland.dm [hunk 1]
```diff
@@ -225,71 +263,60 @@
...
-	name = "slimy cocoon"
-	desc = "Something is moving inside."
-	icon = 'icons/effects/effects.dmi'
-	icon_state = "cocoon_large3"
-	color = rgb(10,120,10)
-	density = TRUE
-	var/hatch_time = 0
-
-/obj/structure/spider/cocoon/abductor/proc/Copy(mob/living/carbon/human/H)
-	var/mob/living/carbon/human/interactive/greytide/clone = new(src)
```
Comment (vuonojenmustaturska) [inline code/modules/antagonists/abductor/equipment/gland.dm]: woop

**Comment by Supermichael777** [comment]
Violence Neutralization:

**Comment by Repukan** [comment]
Why would you remove the funny implants.

**Comment by Time-Green** [comment]
add gib gland

**Pull request merged**